core/validation_schema.py

```diff
@@ -415,7 +415,7 @@
             "Primary Intervention Theme": "str",
             "Single or Multiple Locations": "str",
             "Locations": "str",
-            "Postcodes": "str",
+            "Postcodes": "list",
```

Comment:
Conscious that there are no changes to the validation unit tests... do we want to add some unit tests to cover the new scenarios and code paths introduced with this new type? 

I would usually say that this should be hard requirement for this PR, but given that we may be moving on from this validation code, perhaps it's not as critical

Reply:
I've just realised that validating the type for Postcodes doesn't actually make any sense. If we find that the type is not a list, and that instead it's a string, are we going to return a message to the user saying, "You have entered text instead of a list?". It doesn't make any sense as the application logic is what converts the inputted postcodes from the spreadsheet into a list.

Reply:
I don't think it's even possible for the user to enter a 'wrong type' into the Postcode field as it's generated from the ```extract_postcodes``` method, but if for whatever reason our application side logic fails then the user will be greeted with a rather confusing message.

Reply:
I think we should keep it as a string for validation purposes and convert to an array afterwards. Ultimately, the user is putting 'text' into the field from which the value is generated. It also shares the same cell location as 'Locations', which would make it doubly confusing if there's two types expected in one cell. What if they're both somehow wrong? The user has to put 'text' and a 'list' into the same cell, but only one?